ofnetDns.go

```diff
@@ -0,0 +1,52 @@
+/***
+Copyright 2016 Cisco Systems Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ofnet
+
+import (
+	"errors"
+)
+
+const DnsMaxRespMsgSize = 1024
+
+type NameServer interface {
+	NsLookup([]byte, *string) ([]byte, error)
+}
+
+func processDNSPkt(agent *OfnetAgent, inPort uint32, udpData []byte) ([]byte, error) {
+
+	if agent.nameServer == nil {
+		return nil, errors.New("failed")
```

Comment:
Apart from looking at the stats, I think it's better to have descriptive messages so that we narrow down the issue if anyone is looking at the logs.